main.py

```diff
@@ -104,27 +104,54 @@ def schedule():
 @app.route("/workshops.html")
 def workshops():
     data = _data()
-    data["workshops"] = site_data["workshops"]
+    data["workshops"] = [
+        format_workshop(workshop) for workshop in site_data["workshops"]
+    ]
     return render_template("workshops.html", **data)
 
 
 def format_paper(v):
+    list_keys = ["authors", "keywords", "session"]
+    list_fields = {}
+    for key in list_keys:
+        if isinstance(v.get(key, ""), list):
+            list_fields[key] = v.get(key, "")
+        else:
+            list_fields[key] = v.get(key, "").split("|")
```

Comment:
:nitpick: extract `v.get(key, "")` as a variable

Reply:
oh this is also used in `format_workshop`. Can you make a method so there is less duplicated code?

Reply:
Of course, makes sense!